ckine/figures/figureS6.py

```diff
@@ -0,0 +1,104 @@
+"""
+This creates Figure S6.
+"""
+import string
+import numpy as np
+import pandas as pd
+import seaborn as sns
+import matplotlib.cm as cm
+from .figureCommon import subplotLabel, getSetup, plot_conf_int, import_pMuteins, organize_expr_pred, mutein_scaling, getMutAff
+from ..imports import import_Rexpr, import_samples_2_15, import_pstat
+
+dataMean = import_pMuteins()
+dataMean.reset_index(inplace=True)
+data, _, _ = import_Rexpr()
+data.reset_index(inplace=True)
+unkVec_2_15, _ = import_samples_2_15(N=5)
```

Comment:
You're probably only getting narrow prediction bands because you're only plotting 5 samples. To get an accurate prediction band you'll need ~100 samples. (This might take a long time to run, though.)

Reply:
Try increasing this value.